src/client/user/components/Drawer/ControlPanel/index.tsx

```diff
@@ -125,12 +153,18 @@ export default function ControlPanel() {
   const drawerStates = useDrawerState()
   const drawerIsOpen = drawerStates.controlPanelIsOpen
   const modalDispatch = useDrawerDispatch()
+  const dispatch = useDispatch()
 
   // Fetch short link state and dispatches from redux store through our helper hook.
   const { shortLinkState, shortLinkDispatch } = useShortLink(
     drawerStates.relevantShortLink!,
   )
 
+  // Toggle Link History
+  const isLinkHistoryActive = drawerStates.linkHistoryIsToggled
```

Comment:
Should we change in the drawerStates as well?

Reply:
forgot to inform you, changed this as well!